src/nni_manager/training_service/remote_machine/remoteMachineTrainingService.ts

```diff
@@ -58,8 +58,12 @@ import { mkDirP } from '../../common/utils';
  */
 @component.Singleton
 class RemoteMachineTrainingService implements TrainingService {
-    private machineSSHClientMap: Map<RemoteMachineMeta, Client>;
+    private machineSSHClientMap: Map<RemoteMachineMeta, Client>; //set cluster ssh client
+    private trialSSHClientMap: Map<string, Client>; //set trial ssh client
     private trialJobsMap: Map<string, RemoteMachineTrialJobDetail>;
+    private connectionNumberMap: Map<Client, number>; // set connection number for every ssh client
+    private connectionMetaMap: Map<Client, RemoteMachineMeta>; //set connection meta for every ssh client
+    private maxSSHTrialNumber: number; // every ssh client has a max connection number
```

Comment:
readonly? If yes, suggest to rename the var to UPPER case, like MAX_TRIAL_NUMBER_PER_SSHCONNECTION

Reply:
updated to MAX_TRIAL_NUMBER_PER_SSHCONNECTION